x-pack/solutions/observability/test/functional/apps/infra/home_page.ts

```diff
@@ -171,9 +170,9 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         await pageObjects.infraHome.waitForLoading();
 
         expect(ensureKubernetesTourVisible).to.contain(kubernetesTourText);
+        await pageObjects.infraHome.clickDismissKubernetesTourButton();
```

Comment:
It was like this before I moved this line into `tryForTime` function. I don't remember why I did it, but the test apparently worked fine before that change